WeScan/Edit/EditScanViewController.swift

```diff
@@ -157,7 +157,24 @@ final class EditScanViewController: UIViewController {
         }
         
         let finalImage = fixImageRotation(image: uiImage)
-        let results = ImageScannerResults(originalImage: image, scannedImage: finalImage, detectedRectangle: scaledQuad)
+        
+        /// Prepare the enhanced image using our AdaptiveThresholdCIFilter.
+        let filter = AdaptiveThresholdCIFilter()
```

Comment:
This function is getting really big. 
Maybe we could split it up here, with a simple function that generates the enhanced image?

```
func enhancedImage(for image: UIImage) -> UIImage {
// Implementation
```

Reply:
Fixed in ad8c234